lib/pxutil.js

```diff
@@ -205,6 +206,24 @@ function parseCookieHeader(headerValue) {
     return res;
 }
 
+function accessNestedObjectValueByStringPath(path, obj, separator='.') {
+    var properties = Array.isArray(path) ? path : path.split(separator)
+    return properties.reduce((prev, curr) => prev && prev[curr], obj);
```

Comment:
Fancy! :)
I personally don't like these but it's up to you, I think for loop on the array with the object will be easier to understand, but maybe it is just me.
I still don't understand the '&&'  if you can explain?

Reply:
The && is just to check that the previous object (`prev`) isn't `undefined`. If it is, then we won't try to access a member of it.